src/main/java/com/hms_networks/americas/sc/extensions/string/StringUtils.java

```diff
@@ -78,4 +81,38 @@ public static List split(String string, String delimiter) {
     stringParts.add(endStringPart);
     return stringParts;
   }
+
+  /**
+   * Gets the contents of the supplied {@link InputStream} as a {@link String} in the specified
+   * encoding.
+   *
+   * @param inputStream input stream to read
+   * @param encoding encoding to use
+   * @return string contents of input stream
+   * @throws IOException if an error occurs reading the input stream
+   * @since 1.4.0
+   */
+  public static String getStringFromInputStream(InputStream inputStream, String encoding)
+      throws IOException {
+    // Create output stream for result
+    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+
+    // Create buffer for reading input stream
+    final int bufferSizeBytes = 1024;
+    byte[] buffer = new byte[bufferSizeBytes];
+
+    // Read from input stream and write to output stream
+    final int endOfStreamByte = -1;
```

Comment:
This name is a little misleading as -1 isnt really a byte (even though everything else returned by .read() can be represented as a byte). Maybe "endOfStreamValue" would be more appropriate.